tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -603,22 +603,29 @@ internal static class LoginEvent
 
                     internal const string Failure = Root + ".failure";
                     internal const string FailureUserId = Failure + ".usr.id";
+                    internal const string FailureUserExists = Failure + ".usr.exists";
```

Comment:
Should we move the AppSec tags to a `Tags.Appsec.cs` file and change git ownership?

Reply:
added in 139bd3e25b6c4577effbbe14880302003ad5de8e

Reply:
not sure why I now have DataDog/apm-dotnet as a new required reviewer, is it because I ch anged the CODEOWNERS file? 🤔

Reply:
Nice thanks a lot

Reply:
hmm, I don't know. Maybe?